wrappers/python/python.cpp

```diff
@@ -436,10 +436,8 @@ PYBIND11_MODULE(NAME, m) {
 
     py::class_<rs2::depth_frame, rs2::video_frame> depth_frame(m, "depth_frame");
     depth_frame.def(py::init<rs2::frame>())
-        .def("get_distance", &rs2::depth_frame::get_distance, "x"_a, "y"_a);
-
-
-
+        .def("get_distance", &rs2::depth_frame::get_distance, "x"_a, "y"_a)
+        .def("fit_plane", &rs2::depth_frame::fit_plane, "roi"_a, "iterations"_a, "outliers"_a, "a"_a, "b"_a, "c"_a, "d"_a, "rms"_a);
```

Comment:
Python floats are immutable, so passing by reference for write access doesn't work. This function should use a lambda that returns a tuple with all 6 return args (bool, a, b, c, d, rms)
(See [Pybind11 docs](https://pybind11.readthedocs.io/en/stable/faq.html#limitations-involving-reference-arguments))

Reply:
After sleeping on it, I think it would be best for C++ API as well. It would make sense to add access to plane vertices like we do in DQT, and we certainly don't want to add 12 additional output parameters.
I'll rework the API